druntime/src/rt/lifetime.d

```diff
@@ -1893,7 +1762,13 @@ unittest
     assert(info2.base is carr2.ptr); // no offset, the capacity is small.
 }
 
-unittest
+// I think this whole test is invalid, as is the bug report. Setting internal
```

Comment:
given this, I think this PR (in conjunction with later PRs, you need not do this now) should have a changelog entry and this should be made deprecated.

Reply:
moot now I think. Should be the same behavior.